concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+
+### Backlog
+
+As our project scales, it’s easy to imagine a single unified backlog becoming incredibly cumbersome and hard to manage. Instead, we should adapt our current approach to area-specific boards, which effectively divides our backlog into categories (SGX, SEV,  infrastructure, etc.).
+
+Each area-specific backlog will be housed in a Github project with the following columns:
+
+- Backlog: for issues we want to tackle eventually, but not a high priority
+- Tracking: for issues that are high-priority, and that we want to take action on sooner rather than later. All new project issues should land here by default to ensure they don’t get lost (automatable with stock Github).
+- Active: for issues that are actively being worked on as part of the current sprint.
+- Done: when things are done, they go here (automatable with stock Github). Mostly to ensure the “Active” column stays organized.
+
+In addition to inherently categorizing our backlog, this approach allows new "categories" to be moved around or created as needed -- and, if we decide to use organization-level projects for our sprints, allows the backlog to be bisected across repositories if wanted. This will support modularization as the project grows.
+
+To start, I propose the following backlog projects:
+
+- Documentation
+- Infrastructure
+- SEV
+- SGX
+- WASM
+
+A few things to consider:
+- Downside: issues that don’t fit squarely into a category don’t have a place to go. Maybe a "Miscellaneous" project?
+- Automation opportunity: we could automatically move issues from Tracking to Active when they are assigned to someone. Automatable with Actions.
+
+### Active Sprint Board
+This board is where the team will track progress on current sprint tasks. **It should hold no backlog;** if an issue or PR is on this board, it is being actively worked on.
+
+Here’s how the columns should look:
+
+- Assigned: this indicates a task that is currently being worked on as part of the sprint. Newly-added issues land here by default (automatable with stock Github).
+- Ready for review: where pull requests land by default (automatable with stock Github); indicates it’s ready to be reviewed by someone. Pull requests should move back here from Changes Requested if a fix has been applied and it’s ready for a re-review (not in stock Github, but automatable with Actions).
+- Changes Requested: PRs move here when a review is submitted that requests changes (automatable with stock Github).
+- Approved: PRs that have been approved go here, pending merge (automatable with stock Github).
+- Done: PRs that have been merged, and their associated issues, go here (automatable with stock Github).
```

Comment:
Does "done" on the sprint board automatically move issues to "done" on the backlog boards? Are the issues duplicates of each other? I'm not 100% on how backlog and sprint stay in sync since we don't have pointers on GitHub boards....